acceptance/cluster/localcluster.go

```diff
@@ -580,11 +581,24 @@ func (l *LocalCluster) Start() {
 	}
 }
 
+func (l *LocalCluster) checkConsistency(t *testing.T) {
+	if l.NumNodes() <= 0 {
+		return
+	}
+	// Always connect to the first node in the cluster.
+	kvClient, kvStopper := l.NewClient(t, 0)
+	defer kvStopper.Stop()
+	if pErr := kvClient.CheckConsistency(keys.TableDataMin, keys.TableDataMax); pErr != nil {
```

Comment:
I know it was like this before, but why `TableData{Min,Max}` instead of `Key{Min,Max}`? The system keys should be consistent too.